src/shared/Microsoft.AzureRepos.Tests/AzureReposHostProviderTests.cs

```diff
@@ -171,5 +171,88 @@ public async Task AzureReposProvider_GetCredentialAsync_ReturnsCredential()
             Assert.Equal(personalAccessToken, credential.Password);
             // We don't care about the username value
         }
+
+        [Fact]
+        public async Task AzureReposHostProvider_ConfigureAsync_UseHttpPathSetTrue_DoesNothing()
+        {
+            string key = $"{Constants.GitConfiguration.Credential.SectionName}.https://dev.azure.com.{Constants.GitConfiguration.Credential.UseHttpPath}";
+            var provider = new AzureReposHostProvider(new TestCommandContext());
+
+            var environment = new TestEnvironment();
+            var config = new TestGitConfiguration(new Dictionary<string, IList<string>>
+            {
+                [key] = new List<string> {"true"}
+            });
+
+            await provider.ConfigureAsync(
```

Comment:
Is it easy / worth validating global hasn't been changed?

Reply:
What is "global" ?